README.md

```diff
@@ -42,6 +42,33 @@ module.exports = {
 };
 ```
 
+## Gts/Gjs eslint setup
+
+```js
+// .eslintrc.js
+module.exports = {
+  plugins: ['ember'],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended', // or other configuration
+  ],
```

Comment:
Let's only include the new `overrides` instead of repeating the parts of the config already present above.
```suggestion
```